tools/integration_tests/managed_folders/test_helper.go

```diff
@@ -207,3 +207,30 @@ func listNonEmptyManagedFolders(t *testing.T) {
 		return
 	}
 }
+
+func copyDirAndCheckErr(src, dest string, t *testing.T) {
+	err := operations.CopyDir(src, dest)
+	if err == nil {
+		t.Errorf("Managed folder get copied with view only permission.")
+	}
+
+	operations.CheckErrorForReadOnlyFileSystem(err, t)
+}
+
+func copyObjectAndCheckErr(src, dest string, t *testing.T) {
+	err := operations.CopyObject(src, dest)
+	if err == nil {
+		t.Errorf("Objects in managed folder get copied with view only permission.")
+	}
+
+	operations.CheckErrorForReadOnlyFileSystem(err, t)
+}
+
+func moveAndCheckErr(src, dest string, t *testing.T) {
```

Comment:
Can we rename all the listed methods by adding suffix ForViewPermission, e.g. moveAndCheckErr to moveAndCheckErrForViewPermission?